### PR TITLE
changes

### PR DESCRIPTION
**Pull request by L-BloodStone**

### CODE REVIEW

**Comment by raju5004** [comment]
hello

**Review by L-BloodStone — COMMENTED** [review]
> hello 

Printing $etaNull.

**Comment by raju5004** [comment]
Thik ache guru

File: null.php [hunk 1]
```diff
@@ -8,5 +8,6 @@
 
 if ($etaNull == null){
     $eatNull = "something";
+    echo $eatNull;
```
Comment (L-BloodStone) [inline null.php:11]: printing **$etaNull**.

File: null.php [hunk 2]
```diff
@@ -8,5 +8,6 @@
 
 if ($etaNull == null){
     $eatNull = "something";
```
Comment (raju5004) [inline null.php:10]: ei line ta bujhi nai

Comment (raju5004) [inline null.php:10]: eta bujhi nai

**Review by raju5004 — APPROVED** [review]

File: null.php (hunk 2 above)
Comment (L-BloodStone) [inline null.php:10]: ### whole comment.
image

Comment (L-BloodStone) [inline null.php:10]: 10 line code comment.

**Pull request closed without merging**